core/transport/src/main/proto/grpc/version.proto

```diff
@@ -20,7 +20,7 @@ enum ServiceType {
   METRICS_MASTER_CLIENT_SERVICE = 9;
   JOB_MASTER_CLIENT_SERVICE = 10;
   JOB_MASTER_WORKER_SERVICE = 11;
-  FILE_SYSTEM_WORKER_WORKER_SERVICE = 12;
+  BLOCK_WORKER_CLIENT_SERVICE = 12 [deprecated = true];
```

Comment:
did you intend to mark `FILE_SYSTEM_WORKER_WORKER_SERVICE` deprecated and add a new field? I think for max compatibility we only add new fields instead of replacing them.

Reply:
Does renaming a field, not changing its type or removing it, breaks compatibility? Though it doesn't matter practically for this PR, I have a strong feeling that renaming is the correct thing to do here...